frontend/src/metabase/admin/tasks/containers/Help.jsx

```diff
@@ -99,6 +99,8 @@ export default class Help extends Component {
   render() {
     const { details } = this.state;
     const detailString = JSON.stringify(details, null, 2);
+    const { tag } = MetabaseSettings.get("version");
+    const compactDetailString = JSON.stringify(details);
```

Comment:
`detailString` contains lots of whitespaces that aren't suitable for passing in the URL. From my setup, the `detailString` is about 900+ characters long, and the URL shouldn't exceed around 2000characters.

Reply:
FWIW this idea comes from what we do in the "File a Bug Request" in `Admin -> Troubleshooting`, which passes diagnostics info as a GET parameter. Hopefully trimming whitespace makes it reliably manageable.

Reply:
Wow, thanks for the comment, I think I've missed escaping JSON in the URL. Will fix it ASAP.